diff -urN aaa/gnubiff-1.0.5/configure.ac gnubiff-1.0.5/configure.ac
--- aaa/gnubiff-1.0.5/configure.ac	2003-11-17 16:24:27.000000000 +0100
+++ gnubiff-1.0.5/configure.ac	2003-11-20 16:52:49.000000000 +0100
@@ -65,7 +65,7 @@
 dnl directory; configure checks for this file's existence to make sure
 dnl that the directory that it is told contains the source code in fact
 dnl does.
-AC_CONFIG_SRCDIR([src/gnubiffGTK.cc])
+AC_CONFIG_SRCDIR([src/gnubiff.cc])
 
 dnl syntax: AC_CONFIG_HEADERS (header ..., [cmds], [init-cmds])
 dnl This macro is one of the instantiating macros. Make AC_OUTPUT
