diff -urN aaa/gnubiff-1.0.6/configure.ac gnubiff-1.0.6/configure.ac
--- aaa/gnubiff-1.0.6/configure.ac	2003-11-24 09:49:27.000000000 +0100
+++ gnubiff-1.0.6/configure.ac	2003-12-13 13:28:05.000000000 +0100
@@ -73,7 +73,7 @@
 dnl containing C preprocessor #define statements, and replace `@DEFS@'
 dnl in generated files with `-DHAVE_CONFIG_H' instead of the value of
 dnl DEFS. The usual name for header is `config.h'.
-AM_CONFIG_HEADERS([config/config.h:config/config.h.in])
+AC_CONFIG_HEADERS([config/config.h:config/config.h.in])
 
 dnl Automake generates rules to automatically rebuild `Makefile's,
 dnl `configure', and other derived files like `Makefile.in'.
