diff -urN aaa/gnubiff-1.0.6/src/Biff.cc gnubiff-1.0.6/src/Biff.cc
--- aaa/gnubiff-1.0.6/src/Biff.cc	2003-11-24 09:50:36.000000000 +0100
+++ gnubiff-1.0.6/src/Biff.cc	2003-12-14 00:49:41.000000000 +0100
@@ -454,11 +454,12 @@
       for (gint i=0; i<gint(tmp.size())-1; i+=2) {
 	char c = 0;
 	guint j;
-	for (j=0; j<16; j++)
+	for (j=0; j<16; j++){
 	  if (_table [j] == tmp[i])
 	    c += j*16;
-	  else if (_table [j] == tmp[i+1])
+	  if (_table [j] == tmp[i+1])
 	    c += j;
+	}
 	_password += c;
       }
     }
