diff -urN aa/gnubiff-2.0.1/src/biff.cc gnubiff-2.0.1/src/biff.cc
--- aa/gnubiff-2.0.1/src/biff.cc	2004-09-03 20:19:11.000000000 +0200
+++ gnubiff-2.0.1/src/biff.cc	2004-09-04 14:52:10.583897000 +0200
@@ -68,14 +68,14 @@
 								 GError **error)
 	{
 		BIFF(data)->xml_start_element (context, element_name, attribute_names, attribute_values, error);
-	};
+	}
 
 	void BIFF_xml_error (GMarkupParseContext *context,
 						 GError *error,
 						 gpointer data)
 	{
 		BIFF(data)->xml_error (context, error);
-	};
+	}
 }
 
 
diff -urN aa/gnubiff-2.0.1/src/socket.cc gnubiff-2.0.1/src/socket.cc
--- aa/gnubiff-2.0.1/src/socket.cc	2004-08-31 15:37:52.000000000 +0200
+++ gnubiff-2.0.1/src/socket.cc	2004-09-04 14:52:19.362562440 +0200
@@ -190,7 +190,7 @@
 #endif
 	status_ = SOCKET_STATUS_OK;
 	return 1;
-};
+}
 
 
 gint
